docs/guides/kubernetes.md

```diff
@@ -80,151 +72,92 @@ subjects:
-            - name: COOKIE_SECRET
-              value: "..." # $(head -c32 /dev/urandom | base64 -w 0)
-            - name: IDP_PROVIDER
-              value: google
-            - name: IDP_CLIENT_ID
-              value: "..."
-            - name: IDP_CLIENT_SECRET
-              value: "..."
-            - name: POLICY
-              value: "..." #$(echo "$_policy" | base64 -w 0)
+1. Update your `pomerium-values.yaml` file with the following route:
 
----
-apiVersion: v1
-kind: Service
-metadata:
-  namespace: default
-  name: pomerium
-spec:
-  type: NodePort
-  selector:
-    app: pomerium
-  ports:
-    - port: 30443
-      targetPort: 30443
-      nodePort: 30443
-```
+    ```yaml
+      routes:
+        - from: https://k8s.localhost.pomerium.io
+          to: https://kubernetes
+          allow_spdy: true
+          tls_skip_verify: true
+          kubernetes_service_account_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+          policy:
+            - allow:
+                or:
+                  - domain:
+                      is: pomerium.com
+    ```
 
-Make sure to fill in the appropriate values as indicated.
+    Change the policy to match your configuration.
 
-The policy should be a base64-encoded block of yaml:
+1. Apply the updated values with Helm:
 
-```yaml
-- from: https://k8s.localhost.pomerium.io:30443
-  to: https://kubernetes.default.svc
-  tls_skip_verify: true
-  allow_spdy: true
-  policy:
-    - allow:
-        or:
-          - domain:
-              is: pomerium.com
-  kubernetes_service_account_token: "..." #$(kubectl get secret/"$(kubectl get serviceaccount/pomerium -o json | jq -r '.secrets[0].name')" -o json | jq -r .data.token | base64 -d)
-```
+    ```bash
+    helm upgrade --install pomerium pomerium/pomerium --values pomerium-values.yaml
+    ```
 
-Applying this configuration will create a Pomerium deployment and service within kubernetes that is accessible from `*.localhost.pomerium.io:30443`.
+    This will create a Pomerium route within kubernetes that is accessible from `*.localhost.pomerium.io`.
 
-## Kubectl
+## Configure Kubectl
 
-Pomerium uses a custom Kubernetes exec-credential provider for kubectl access. This provider will open up a browser window to the Pomerium authenticate service and generate an authorization token that will be used for Kubernetes API calls.
+The [pomerium-cli] tool can be used by kubectl as an auth helper. Once configured, connections to the cluster will open a browser window to the Pomerium authenticate service and generate an authorization token that will be used for Kubernetes API calls.
 
-The Pomerium Kubernetes exec-credential provider can be installed via go-get:
+In addition to the methods listed in the page linked above, `pomerium-cli` can be installed via go-get:
 
 ```bash
 env GO111MODULE=on GOBIN=$HOME/bin go get github.com/pomerium/pomerium/cmd/pomerium-cli@master
 ```
 
 Make sure `$HOME/bin` is on your path.
```

Comment:
Outstanding questions:

1. Should this section still be included, and/or should it be moved to the [Downloads page section](https://pomerium.com/docs/releases.html#pomerium-cli) for pomerium-cli?
2. If yes to either, should it be adjusted to point at the latest release branch instead of `master`, then updated as new major releases are cut?

Reply:
I think just link to download/install section.  This was probably before we had a fully baked process for cli install.

Reply:
```suggestion
```